_posts/2018-12-13-arms-final-report.md

```diff
@@ -0,0 +1,151 @@
+---
+layout: post
+title:  "ARMS CSC 596 Final Report"
+categories: ["Missouri State University", "ARCS"]
+tags: ["self-study", "csc596", "robotics", "python", "proposal", ode, sdf]
```

Comment:
Put the last two in quotes, or remove quotes from all.

Reply:
Done